DependencyInjection/Compiler/AspectCollectorPass.php

```diff
@@ -19,16 +19,15 @@
  */
 class AspectCollectorPass implements CompilerPassInterface
 {
-
     /**
-     * You can modify the container here before it is dumped to PHP code.
-     *
-     * @param ContainerBuilder $container
-     *
-     * @api
+     * {@inheritdoc}
      */
     public function process(ContainerBuilder $container)
     {
+        if (!$container->hasDefinition('goaop.aspect.container')) {
```

Comment:
see: https://github.com/matthiasnoback/SymfonyDependencyInjectionTest#standard-test-for-unobtrusiveness

It is a standard, recommended way to write compiler pass.